<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client</class>
 <widget class="QDialog" name="client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1229</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(243, 243, 243);</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>1141</width>
     <height>551</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Clients</string>
    </attribute>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>761</width>
       <height>391</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(27, 75, 101);</string>
      </property>
      <property name="text">
       <string>Liste des Clients</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_pdfC">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	
	
	border-image: url(:/Downloads/pdf (1).png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="rechercherC">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>30</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border:none;
border-radius:5px;
color: rgb(27, 75, 101);
</string>
      </property>
      <property name="placeholderText">
       <string>rechercher</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_16">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>40</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	
	border-image: url(:/Downloads/loupe.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableView" name="tab_clients">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>641</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>351</width>
       <height>391</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton #pushButton{
	
	background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton #pushButton:hover{
	
	background-color: rgb(98, 185, 203);
}
QPushButton #pushButtonp:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color: rgb(98, 185, 203);
}</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>311</width>
        <height>351</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius:20px;

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(27, 75, 101);
</string>
      </property>
      <property name="text">
       <string>Formulaire</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(27, 75, 101);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>110</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(27, 75, 101);</string>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);color: rgb(27, 75, 101);</string>
      </property>
      <property name="text">
       <string>Adresse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>180</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);color: rgb(27, 75, 101);</string>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>250</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);color: rgb(27, 75, 101);</string>
      </property>
      <property name="text">
       <string>Phone</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nomc">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
border:none;
border-radius:5px;
color: black</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_ajouterC">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>320</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-radius:5px;
   border-right:1px solid #aaaaaa;
	border-bottom:1px solid #aaaaaa;
}
QPushButton:enabled{
background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(190, 233, 231);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton:disabled{
background-color: #aaaaaa;
	color: rgb(255, 255, 255);
}

</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_modififerC">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>320</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-radius:5px;
   border-right:1px solid #aaaaaa;
	border-bottom:1px solid #aaaaaa;
}
QPushButton:enabled{
background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(190, 233, 231);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton:disabled{
background-color: #aaaaaa;
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cinc">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>130</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
border:none;
border-radius:5px;
color: black</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="adressec">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>200</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
border:none;
border-radius:5px;
color: black</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="emailc">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>200</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
border:none;
border-radius:5px;
color: black</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="phonec">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>270</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
border:none;
border-radius:5px;
color: black</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cinc_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>450</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
border:none;
border-radius:5px;
color: black</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sexec">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>270</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
border:none;
border-radius:5px;
color: black</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>250</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(27, 75, 101);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Sexe</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>420</y>
       <width>431</width>
       <height>51</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	
	qproperty-icon: url(:/Downloads/trier.png);
	qproperty-iconSize: 30px 30 px;
text-align:center;
	color: rgb(27, 75, 101);
}</string>
      </property>
      <property name="text">
       <string>Trier par</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_trierC">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-radius:5px;
   border-right:1px solid #aaaaaa;
	border-bottom:1px solid #aaaaaa;
}
QPushButton:enabled{
background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(190, 233, 231);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton:disabled{
background-color: #aaaaaa;
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>Trier</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_tri">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(27, 75, 101);</string>
      </property>
      <item>
       <property name="text">
        <string>ALL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>par CIN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>par Nom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>par Adresse</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>430</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-radius:5px;
   border-right:1px solid #aaaaaa;
	border-bottom:1px solid #aaaaaa;
}
QPushButton:enabled{
background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(190, 233, 231);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton:disabled{
background-color: #aaaaaa;
	color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>chatbox</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>420</y>
       <width>311</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius:20px;

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_suppc">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>450</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-image: url(:/Downloads/bouton-supprimer.png);
color:white;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>430</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(27, 75, 101);</string>
     </property>
     <property name="text">
      <string>CIN</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_suppC">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>450</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>420</y>
       <width>231</width>
       <height>80</height>
      </rect>
     </property>
     <widget class="QPushButton" name="pb_ONC">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-radius:5px;
   border-right:1px solid #aaaaaa;
	border-bottom:1px solid #aaaaaa;
}
QPushButton:enabled{
background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(190, 233, 231);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton:disabled{
background-color: #aaaaaa;
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>ON</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_OFFC">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-radius:5px;
   border-right:1px solid #aaaaaa;
	border-bottom:1px solid #aaaaaa;
}
QPushButton:enabled{
background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(190, 233, 231);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton:disabled{
background-color: #aaaaaa;
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>OFF</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(27, 75, 101);
</string>
      </property>
      <property name="text">
       <string>Buzzer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(27, 75, 101);
</string>
      </property>
      <property name="text">
       <string>Etat :</string>
      </property>
     </widget>
     <widget class="QLabel" name="etatardC">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(27, 75, 101);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="chatbox">
    <attribute name="title">
     <string>Histrorique</string>
    </attribute>
    <widget class="QPushButton" name="pb_hisC">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>100</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-radius:5px;
   border-right:1px solid #aaaaaa;
	border-bottom:1px solid #aaaaaa;
}
QPushButton:enabled{
background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(190, 233, 231);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton:disabled{
background-color: #aaaaaa;
	color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>valider</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="history">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>100</y>
       <width>571</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Statistique</string>
    </attribute>
    <widget class="QFrame" name="horizontalFrame">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>771</width>
       <height>481</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </widget>
    <widget class="QPushButton" name="pb_statC">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-radius:5px;
   border-right:1px solid #aaaaaa;
	border-bottom:1px solid #aaaaaa;
}
QPushButton:enabled{
background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(190, 233, 231);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton:disabled{
background-color: #aaaaaa;
	color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>stat</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
